src/main/java/org/reactome/release/orthopairs/Main.java

```diff
@@ -53,8 +33,9 @@
      */
 
     private static final Logger logger = LogManager.getLogger();
+    private static final String pantherQfOFilename = "QfO_Genome_Orthologs.tar.gz";
+    private static final String pantherHCOPFilename = "Orthologs_HCOP.tar.gz";
```

Comment:
Should file names come from configuration?

Reply:
They've never changed so I thought it would be ok to have them as constants, but I can revert that